### DISCUSSION TITLE
posts/029-the-art-path/

### DISCUSSION THREAD

**Discussion by giscus** [Yamabuki-bakery/tasty-discussion#16] [Announcements]
# posts/029-the-art-path/

在把畫猜當作目標的兩年之後，俺也許要計劃一個新旅程了

**Comment by Parsifa1** [comment]
加油 希望我也有一天能有机会重新拾起画笔😢

**Reply by PermeationLoop** [reply]
俺衷心希望妳有一天能重拾畫筆……

**Comment by Sayafx** [comment]
**画画难以一步登天**

当我略微有一些画画经验后，作画时就会无意识地考虑很多东西。为什么说是无意识呢，因为每当我尝试讲解作画流程时，很难解释清楚为什么这一步要这么画。因此说画画是靠不断积累进步的，一旦开始画画，就会无意识调用曾经积累的绘画经验和知识。而且以我个人的经验看，练习时通常感觉不到很大进步，往往是“放下画笔”之后，通过“偶尔的作画”不经意间发现好像可以画得更好了。有些像我之前读高中的时候老师说的，学习卡住了先去学别的东西，再回过头来看就会变得容易。

**练习方向**

有关练习的方向实在是太多，虽然可以像 Krenz 一样大致分为3个科目，透视、构成、色彩，但其实三者之间还牵连很深，互相牵扯。举一些例子。

* 透视：以 3D 的方式来思考画画，如果能在脑内对一个 3D 物体转动任意角度并 Print 到画布上，那么理论上就可以画几乎所有物体。然而实操时会面临问题：画某个物体需要很严谨的运算，如果画的物件过多或者主题过于复杂，大脑容易过载。同时会发现即使透视画的正确，它可能只是“正确”地存在于那里，但是形状不够好看，明明已经很严谨的计算过了……单纯通过使用透视技法或 3D 辅助画场景，很容易陷入构图困境，而且透视大法面对那些某些偏平面的画风又貌似失灵了。

* 构成和色彩：色彩离不开明暗交界线和阴影形状，形状的好看与否其实和构成有很大关系，同时考虑到用 明暗交界线/色块 的形状做视线引导，也涉及到了构成的知识。

* 构成和透视：构成把物体的做形状好看，除非画卡通类的，其他的总要有体积效果，这也牵扯到了透视问题……

得出结论：单方向的堆量训练很难短期得到很大的回报，需要和其他方面的经验融合。
因此我觉得训练有侧重就好，然后比如画构成涉及到一些透视，把透视部分尽力做到完美就好。我的想法是，训练不用太过僵化，只要**正确**认识到自己薄弱的地方然后做一些训练就好。Krenz 在他的3门正课里讲了针对不同方向的很多练习方式，我就不K门弄斧了。

我现在非常佛系，只是偶尔画一画，已经不苛求自己专门做训练了。

**现在画画的动机**

* 我现在对制作无料很感兴趣，因此现在学画的一大动机是画出精美插画或立绘，足以制作明信片\徽章\立牌，曾经幻想过自己画一堆美少女立绘，做成立牌排上一桌子。
* 偶尔会想和社团的朋友一起做拜年祭节目，我负责一部分绘画。
* 现在还是很想做一个 Galgame Demo ，人物设计、美少女立绘、场景图片是必不可少的。

然而现在分配给画画的时间也少的可怜……

**Reply by PermeationLoop** [reply]
感謝回覆，，，

 > 通过“偶尔的作画”不经意间发现好像可以画得更好了

這是真的，俺現在回過頭去看我以前的畫，確實會覺得當時的筆觸顯得稚嫩，，，


 > 透视：以 3D 的方式来思考画画

能夠計算 3D 確實能在一些困難的場景下起到輔助的作用，就能夠不需要去找精確的參考了；俺在理論上瞭解，但是實踐不足，也並不想去特地練習（會很枯燥）所以俺還是對構成課更感興趣一點．

 > 畫畫的動機

製作週邊立牌確實是一個好的激勵方向，俺也認爲把實物留存在別人手中很有意義，只是這種機會並不多（俺這邊也沒有淘寶上面那種簡單就能定製的店家，，，我想過要是有過塑機就可以用普通的打印紙做書籤🔖了，可惜沒有

剩下的兩個就很考團隊合作了，以俺之前做漢化的經驗來看，在小團隊裏做這些難度會相當🆙️🆙️，，，

所以現在我在想，在不知道方向在哪的時候，放空腦袋無腦地去做 K 課程的作業也不失爲一種好的選擇，，，
